nano/node/json_handler.cpp

```diff
@@ -2818,6 +2818,7 @@ void nano::json_handler::payment_wait ()
 void nano::json_handler::process ()
 {
 	const bool json_block_l = request.get<bool> ("json_block", false);
+	const bool watch_work_l = request.get<bool> ("watch_work", true);
```

Comment:
Probably this option requires protection with enable_control? Same as work_generate

Reply:
handled same way as process force